system-tests/_scripts/launch-cluster.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+if [ -z "${INSTALLER_URL}" ]; then
```

Comment:
What is `INSTALLER_URL` and where do I take it?

Reply:
It's the installer dowload url you find on https://dcos.io/releases/ e.g., `https://downloads.dcos.io/dcos/stable/dcos_generate_config.sh`